qiskit/visualization/gate_map.py

```diff
@@ -1122,7 +1122,13 @@ def plot_circuit_layout(circuit, backend, view="virtual", qubit_coordinates=None
     Args:
         circuit (QuantumCircuit): Input quantum circuit.
         backend (Backend): Target backend.
-        view (str): Layout view: either 'virtual' or 'physical'.
+        view (str): How to label qubits in the layout. Options:
+
+          - ``"virtual"``: Label each qubit with the name of the virtual qubit that
+            mapped to it.
+          - ``"physical"``: Label each qubit with the name of the physical qubit that it
+            corresponds to on the device.
```

Comment:
Are they names or indices? I'm wondering if people would read this and think it would say `q_reg[12]` or something.

Reply:
I went back and forth about whether "name" or "index" would be more clear. I agree with you now, I'll change it to "index"